serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc

```diff
@@ -0,0 +1,388 @@
+= Timeouts on events for {context}
+:compat-mode!:
+// Metadata:
+:description: Using timeouts in {context}
+:keywords: kogito, workflow, serverless, timeout, timer, expiration
+
+When you define a state in a serverless workflow, you can use the `timeouts` property to configure the maximum time to complete this state.
+When that time is overdue, the state is considered timed-out, and the engine continues the execution from this state. The execution flow depends on the state type, for instance,
+a transition to a next state.
+All the properties you can use to configure state timeouts are described in the link:{spec_doc_url}#event-timeout-definition[Serverless Workflow specification].
+
+Event-based states can use the sub-property `eventTimeout` to configure the maximum time to wait for an event to arrive.
+This property uses `ISO 8601` data and time standard to specify a duration of time. It follows the format `PnDTnHnMn.nS` with days considered to be exactly 24 hours. For instance,
+`PT15M` configures 15 minutes, and `P2DT3H4M` defines 2 days, 3 hours and 4 minutes.
+
+[NOTE]
+====
+Event timeouts can not be defined as a specific point in time, but it should be an amount of time, a duration, which is considered to start when the referred state becomes active in the workflow.
+====
+
+[IMPORTANT]
+====
+{product_name} currently, has timeouts support only for *Callback* and *Switch* states with events. Other states will be included in the future releases.
```

Comment:
Please add this information to the CNCF guide.

Reply:
+1